<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrystalGrowthTrackerMain</class>
 <widget class="QMainWindow" name="CrystalGrowthTrackerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CrystalGrowthTracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Source</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QScrollArea" name="_sourceScrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>304</width>
               <height>110</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_viewSubButton">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Zoom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="_sourceZoomSpinBox">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>5.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Subimages</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="_subScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>625</width>
             <height>105</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="_subimageComboBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="_detectButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Zoom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="_subimageZoomSpinBox">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>4.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="_exitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open </string>
    </property>
    <addaction name="_actionOpen"/>
    <addaction name="_actionSaveSubimages"/>
    <addaction name="_actionSaveCurrentSubimage"/>
    <addaction name="separator"/>
    <addaction name="_actionExit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="_actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="_actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="_actionSaveSubimages">
   <property name="text">
    <string>Save Subimages</string>
   </property>
  </action>
  <action name="_actionSaveCurrentSubimage">
   <property name="text">
    <string>Save Subimage</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_exitButton</sender>
   <signal>pressed()</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>953</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_detectButton</sender>
   <signal>pressed()</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>feature_detect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>load_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_actionSaveSubimages</sender>
   <signal>triggered()</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>save_subimages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_sourceZoomSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>source_zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_subimageZoomSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>subimage_zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>966</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_subimageComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>display_subimage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_actionSaveCurrentSubimage</sender>
   <signal>triggered()</signal>
   <receiver>CrystalGrowthTrackerMain</receiver>
   <slot>save_current_subimage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
